src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
@EricDunsworth the extend ```%noscript-display-block-important``` is only used by the menu, lightbox and overlay. Considering that your fix is limited to the menu, then I adding the ```media screen``` instruction directly in the menu sass is the best solution. 

Sorry but am not able to see how creating a new extend in the global noscript that only apply to menu is usefull. Are you planning to modify that new extend or re-use it in an another theme, like GCweb?